bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gmos/GmosRule.java

```diff
@@ -1850,6 +1855,33 @@ public IConfigMatcher getMatcher() {
     };
 
     /**
+     * REL-2456: We request a Phase II check to give a warning if the DTA-X
+     * position is not a multiple of the y-pixel binning.
+     */
+    private static IConfigRule DTA_X_Y_MULTIPLE_BINNING_RULE = new IConfigRule() {
+        private static final String errMsg = "The DTA X offset (%d) should be multiple of the Y-binning (%d)";
+
```

Comment:
Apropos of one of my comments below, for example here:

````java
    public static final String Id = PREFIX + "DTA_X_Y_MULTIPLE_BINNING_RULE"
````

and just make the entire `DTA_X_Y_MULTIPLE_BINNING_RULE` public too.  Then you can reference it with: `DTA_X_Y_MULTIPLE_BINNING_RULE.Id`.